src/test/java/bio/terra/service/resourcemanagement/azure/AzureResourceConfigurationTest.java

```diff
@@ -271,6 +276,21 @@ public void testAbilityToDeployApplication() {
 
           // Perform file operations
           createFileAndSign(fileSystemClient);
+
+          // Create a table service client by authenticating using the found key
+          TableServiceClient tableServiceClient =
+              new TableServiceClientBuilder()
+                  .credential(new AzureNamedKeyCredential(storageAccountName, key))
+                  .endpoint("https://" + storageAccountName + ".table.core.windows.net")
+                  .buildClient();
+          TableClient tableClient = tableServiceClient.createTable("files");
+          String datasetId = UUID.randomUUID().toString();
+          String fileId = UUID.randomUUID().toString();
+          TableEntity entity =
+              new TableEntity(datasetId, fileId)
+                  .addProperty("fileId", fileId)
+                  .addProperty("description", "A test table entry");
+          tableClient.createEntity(entity);
```

Comment:
is there a way to demonstrate this with a transsaction?

Reply:
I think `TableTransactionAction` will let you do this

Reply:
Good idea!